gradle/wrapper/gradle-wrapper.properties

```diff
@@ -19,4 +19,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
O que acha de já colocar a versão mais recente?
distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-all.zip

Reply:
Please always communicate in english.